pending/xnviewmp.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2018 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-19
+
+-->
+<cleaner id="xnviewmp">
```

Comment:
```suggestion
<cleaner id="xnviewmp" os="windows">
```

Reply:
Seems I didn't saw that... :-/

Reply:
Maybe you didn't see the comment, but it looks like you made the change anyway, so that's good

https://github.com/az0/cleanerml/pull/147/files#diff-74849e7d8c5fa738b93824b14441897bR36

Reply:
Yeah, I'm getting better and better with every PR... ;-)

Reply:
Yes, I noticed. :) Practice makes perfect